koku/masu/external/kafka_msg_handler.py

```diff
@@ -70,18 +70,21 @@ class KafkaMsgHandlerError(Exception):
     """Kafka msg handler error."""
 
 
-def divide_csv_daily(file_path: os.PathLike, manifest_id: int):
-    """
-    Split local file into daily content.
-    """
-    daily_files = []
-
+def get_data_frame(file_path: os.PathLike):
+    """read csv file into dataframe"""
     try:
-        data_frame = pd.read_csv(file_path, dtype=pd.StringDtype(storage="pyarrow"))
+        return pd.read_csv(file_path, dtype=pd.StringDtype(storage="pyarrow"), on_bad_lines="warn")
     except Exception as error:
```

Comment:
This should be more targeted unless there is a very good reason for catching all exception types. Catching `pandas.errors.ParserError` and maybe `UnicodeDecodeError` would be better.

Reply:
This exception is not the focus of this PR. If you feel it should be more targeted, then let's raise a tech-debt issue to address it.